pkg/oidctest/issuer.go

```diff
@@ -85,6 +87,7 @@ func NewIssuer(t *testing.T) (jose.Signer, string) {
 		redirectURL, err := url.Parse(r.URL.Query().Get("redirect_uri"))
 		if err != nil {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
```

Comment:
This was a bug, the rest I added for completeness.

Reply:
Thank you! I keep forgetting these darn returs